docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  jenkins:
+    image: jenkins/jenkins:lts
+    user: 1000:1000 #set user to root to avoid permission issues
+    ports:
+      - 8080:8080 #expose port 8080 to host
+    volumes:
+      - JENKINS_DATA:/var/jenkins_home #mount volume to persist data
+      - ./jobs/RCW:/var/jenkins_home/jobs/RCW 
```

Comment:
You will have to mount the docker socket of host as well, other wise build stage will faiil as it requires docker to build images.

Reply:
Okay!